src/components/forms/inputs/dropdown/dropdown.tsx

```diff
@@ -1,55 +1,69 @@
-import { useState } from 'react';
-import tw, { styled } from 'twin.macro';
+import { useLayoutEffect, useRef, useState } from 'react';
+import 'twin.macro'
 
-import { Icons } from '../../../../components/icon';
+import { Icons } from '../../../icon/index';
 import { useClickOutside } from '../../../../hooks';
-import { DropdownProps } from './dropdown-types';
 
-export const Dropdown = ({ items, value, onChange }: DropdownProps) => {
-  const [isExpanded, setIsExpanded] = useState(false);
-  const node = useClickOutside(() => setIsExpanded(false));
-  const selectedValue = items.find((item) => value === item.value);
+type Position = 'top' | 'bottom';
+
+interface Item {
+  value: string;
+  label: React.ReactNode;
+}
+
+interface Props {
+  items: Item[];
+  action: (value: string) => void;
+  selectedValue: string;
```

Comment:
type Value = string | number;
```suggestion
  selectedValue: Value;
```